integration/kube_integration_test.go

```diff
@@ -163,6 +163,7 @@ func TestKube(t *testing.T) {
 	t.Run("TrustedClustersSNI", suite.bind(testKubeTrustedClustersSNI))
 	t.Run("Disconnect", suite.bind(testKubeDisconnect))
 	t.Run("Join", suite.bind(testKubeJoin))
+	t.Run("ConnectionLimit", suite.bind(testKubeConnectionLimit))
```

Comment:
```suggestion
	t.Run("KubeConnectionLimit", suite.bind(testKubeConnectionLimit))
```

Reply:
I'm just following the existing naming scheme here, including the kube prefix makes little sense when we're running these as subtests of a kube test.

Reply:
Whoops, just noticed that this is already `TestKube` 😅